packages/flame/src/Toaster/story.tsx

```diff
@@ -0,0 +1,179 @@
+            },
+          });
+        }}
+      >
+        Generate success
+      </Button>
+      <Button
+        type="button"
+        onClick={() => {
+          addActionableToast(
+            {
+              content: `I have an action!`,
+              actionTitle: 'Undo',
+              actionCallback: () => {
+                // eslint-disable-next-line no-console
+                console.log('You clicked the undo button!');
+              },
+            },
+            {
+              appearance: 'error',
+            },
+          );
+        }}
+      >
+        Generate error
+      </Button>
+    </Spacer>
+  );
+};
+
+const LongToastExample = () => {
+  const { addToast } = useToasts();
+
+  return (
+    <Spacer>
+      <Button
+        type="button"
+        onClick={() => {
+          addToast('This toast has a much more longer message than the rest of the toasters.');
+        }}
+      >
+        Generate success
+      </Button>
+      <Button
+        type="button"
+        onClick={() => {
+          addToast('This toast has a much more longer message than the rest of the toasters.', {
+            appearance: 'error',
+          });
+        }}
+      >
+        Generate error
+      </Button>
+    </Spacer>
+  );
+};
+
+stories.add(
+  'Toaster',
+  () => (
+    <ToasterProvider>
+      <div>
+        <Heading2>A toast will auto dismiss itself</Heading2>
+        <AutoDismissToastExample />
+      </div>
+      <div>
+        <Heading2>A toast with an action will have a progress bar</Heading2>
+        <ActionableToastExample />
+      </div>
+      <div>
+        <Heading2>Create a toast with a long message</Heading2>
+        <LongToastExample />
+      </div>
+    </ToasterProvider>
+  ),
+  { percy: { skip: true } },
+);
+
+stories.add('percy snapshots', () => {
+  const commonProps = {
+    autoDismiss: false,
+    autoDismissTimeout: 1000,
+    transitionDuration: 1000,
+    isRunning: false,
+    onDismiss: noop,
+    onMouseEnter: noop,
+    onMouseLeave: noop,
+    placement: 'bottom-center' as 'bottom-center',
+    transitionState: 'entered' as 'entered',
+  };
+
+  return (
+    <ToasterProvider>
+      <div>
+        <Toaster appearance="success" {...commonProps}>
+          This is a success toaster
+        </Toaster>
+
+        <Toaster appearance="error" {...commonProps}>
+          This is a failure toaster
+        </Toaster>
+
+        <Toaster appearance="success" {...commonProps} autoDismiss>
+          <ActionableToastContent actionCallback={noop} actionTitle="Undo">
+            This is a success toaster with action
+          </ActionableToastContent>
+        </Toaster>
+
+        <Toaster appearance="error" {...commonProps} autoDismiss>
+          <ActionableToastContent actionCallback={noop} actionTitle="Undo">
+            This is a success toaster with action
+          </ActionableToastContent>
+        </Toaster>
+      </div>
+    </ToasterProvider>
+  );
+});
```

Comment:
Is it possible to capture a Toaster with a (static) progress state for Percy?

Reply:
Not unless I remove animations entirely from the component and change a chunk of the underlying implementation just to have visual diffing working.

Not worth it.

Reply:
Giving it a spin to see [how Percy behaves](https://docs.percy.io/docs/animations#automatic-animation-freezing-in-percy) by enabling progress on the actionable examples.

Reply:
No dice - oh well, let's keep as is for now :+1: